providers/implementations/kdfs/pbkdf2.c

```diff
@@ -179,9 +202,15 @@ static int pbkdf2_set_membuf(unsigned char **buffer, size_t *buflen,
 }
 
 static int pbkdf2_lower_bound_check_passed(int saltlen, uint64_t iter,
-                                           size_t keylen, int *error,
-                                           const char **desc)
+                                           size_t keylen, size_t passlen,
+                                           int *error, const char **desc)
 {
+    if ((passlen * 8) < KDF_PBKDF2_MIN_PASSWORD_LEN) {
```

Comment:
If the unit of `KDF_PBKDF2_MIN_PASSWORD_LEN` is bytes, we should change `(passlen * 8)` to `passlen`?

Reply:
C pre processor macros are source code text without a type.

Reply:
Indeed, there is no type info for C macro.

However, in the `pbkdf2_derive` function, the `passlen`  passed to the `pbkdf2_lower_bound_check_passed` function is a number of bytes.

Therefore, `(passlen * 8) < KDF_PBKDF2_MIN_PASSWORD_LEN` seems to be always true unless `passlen` is zero.